app/helpers/displayHelpers.php

```diff
@@ -3776,7 +3776,7 @@ function caEditorBundleShowHideControl($po_request, $ps_id_prefix, $pa_settings=
 		$ps_preview_id_prefix = preg_replace("/[0-9]+\_rel/", "", $ps_id_prefix);
 
 		$vs_buf  = "<span class='bundleContentPreview' id='{$ps_preview_id_prefix}_BundleContentPreview'>{$ps_preview_init}</span>";
-		$vs_buf .= "<span style='position: absolute; top: 2px; right: 7px;'>";
+		$vs_buf .= "<span class='bundleShowHideControl'>";
```

Comment:
This is a change to a core (non-theme) file.  Perhaps we can push this upstream and put their stock style into their css file.